pyanaconda/dbus_addons/baz/__main__.py

```diff
@@ -1,4 +1,5 @@
 from pyanaconda.dbus_addons.baz.baz import Baz
 
-module = Baz()
-module.run()
+if __name__ == "__main__":
```

Comment:
This is not necessary here: https://docs.python.org/3.6/library/__main__.html

Reply:
Hmm, good to know. I'll remove this part then.